rand() and uniform_() for complex dtype

pbelevich opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#35585 rand() and uniform_() for complex dtype**

Differential Revision: [D20820378](https://our.internmc.facebook.com/intern/diff/D20820378)

pbelevich committed:
rand() and uniform_() for complex dtype

[ghstack-poisoned]

pbelevich committed:
Update on "rand() and uniform_() for complex dtype"




[ghstack-poisoned]

pbelevich committed:
Update on "rand() and uniform_() for complex dtype"




[ghstack-poisoned]

pbelevich committed:
Update on "rand() and uniform_() for complex dtype"




[ghstack-poisoned]

pbelevich committed:
Update on "rand() and uniform_() for complex dtype"




[ghstack-poisoned]

pbelevich committed:
Update on "rand() and uniform_() for complex dtype"




[ghstack-poisoned]

pbelevich committed:
Update on "rand() and uniform_() for complex dtype"




[ghstack-poisoned]

pbelevich committed:
Update on "rand() and uniform_() for complex dtype"




[ghstack-poisoned]

pbelevich committed:
Update on "rand() and uniform_() for complex dtype"




[ghstack-poisoned]

pbelevich committed:
Update on "rand() and uniform_() for complex dtype"




[ghstack-poisoned]

pbelevich committed:
Update on "rand() and uniform_() for complex dtype"




[ghstack-poisoned]

pbelevich committed:
Update on "rand() and uniform_() for complex dtype"




[ghstack-poisoned]

pbelevich committed:
Update on "rand() and uniform_() for complex dtype"




[ghstack-poisoned]

pbelevich committed:
Update on "rand() and uniform_() for complex dtype"




[ghstack-poisoned]

pbelevich committed:
Update on "rand() and uniform_() for complex dtype"




[ghstack-poisoned]

pbelevich committed:
Update on "rand() and uniform_() for complex dtype"




[ghstack-poisoned]

pbelevich committed:
Update on "rand() and uniform_() for complex dtype"


Differential Revision: [D20820378](https://our.internmc.facebook.com/intern/diff/D20820378)

[ghstack-poisoned]

pbelevich committed:
Update on "rand() and uniform_() for complex dtype"


Differential Revision: [D20820378](https://our.internmc.facebook.com/intern/diff/D20820378)

[ghstack-poisoned]

anjali411 reviewed (approved):
Thanks for the PR @pbelevich :D

pbelevich committed:
Update on "rand() and uniform_() for complex dtype"


Differential Revision: [D20820378](https://our.internmc.facebook.com/intern/diff/D20820378)

[ghstack-poisoned]

pbelevich committed:
Update on "rand() and uniform_() for complex dtype"


Differential Revision: [D20820378](https://our.internmc.facebook.com/intern/diff/D20820378)

[ghstack-poisoned]
